otter_welcome_buddy/cogs/hiring_timelines.py

```diff
@@ -1,59 +1,77 @@
 import os
 
 from apscheduler.schedulers.asyncio import AsyncIOScheduler
+from discord import TextChannel
 from discord.ext import commands
+from discord.ext.commands import Bot
+from discord.ext.commands import Context
 
 from otter_welcome_buddy.common.constants import CronExpressions
 from otter_welcome_buddy.common.utils.dates import DateUtils
+from otter_welcome_buddy.common.utils.types.common import DiscordChannelType
 from otter_welcome_buddy.formatters import timeline
 from otter_welcome_buddy.log import wrapper
 
 
 class Timelines(commands.Cog):
     """Hiring events for every month"""
 
-    def __init__(self, bot, messages_formatter):
-        self.bot = bot
-        self.messages_formatter = messages_formatter
+    def __init__(self, bot: Bot, messages_formatter: type[timeline.Formatter]):
+        self.bot: Bot = bot
+        self.messages_formatter: type[timeline.Formatter] = messages_formatter
         self.scheduler: AsyncIOScheduler = AsyncIOScheduler()
+
         self.__configure_scheduler()
 
-    # Commented the !start !stop since I always forget to turn on the trigger
-    # @commands.command()
-    # async def start(self, _):
-    # """Command to interact with the bot and start cron"""
-    # self.__configure_scheduler()
+    @commands.group(
+        brief="Commands related to Timelines messages!",
+        invoke_without_command=True,
+        pass_context=True,
+    )
+    async def timelines(self, ctx: Context) -> None:
+        """
+        Timelines will help to keep track of important events to be announced via cronjobs
+        """
+        await ctx.send_help(ctx.command)
+
+    @timelines.command(brief="Start the cronjob for the timeline messages")  # type: ignore
+    async def start(self, _: Context) -> None:
```

Comment:
Can we include documentation in how to invoke it? Like !start? I guess it's the correct way, this comment applies for all commands.